cli/bzlmod/bzlmod.go

```diff
@@ -0,0 +1,70 @@
+package bzlmod
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/bazelisk"
+)
+
+var (
+	once       sync.Once
+	useModules bool
+	infoErr    error
+)
+
+func UseModules() (bool, error) {
```

Comment:
nit: would it make sense to call this `Enabled()`? so the callsite is `if bzlmod.Enabled() { ... }`

Reply:
It's a lot nicer with the new name ☺️